### DISCUSSION TITLE
Notification module

### DISCUSSION THREAD

**Discussion by LeVraiArdox** [Amitabha37377/Dotfiles#4] [Q&A]
Hi there. I just wanted to ask, what module did you used for the notification popup ? Is it home-made (if yes, what is the file), or you used a module that already exist

**Comment by Amitabha37377** [comment]
The notifications are from awesomewm'w builtin notification daemon `naughty`. My configuration for that is in `deco/notifications.lua` file.
For the notification center, the code is in `popups/notif_center/main.lua` file

**Reply by LeVraiArdox** [reply]
Oh, ok, so i don't understand why i have a random popup instead of my notifs, since i tested your rice and changed it after... I thought it could be in another config folder that i forget to check.
